eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/CDDL.hs

```diff
@@ -250,7 +268,7 @@ update_committee =
     =:~ grp
       [ 4
       , a $ gov_action_id / VNil
-      , a (set committee_cold_credential)
+      , a (tagged_set committee_cold_credential)
```

Comment:
I believe this was wrong. We do use the same Set decoder for this action as we do everywhere else, so this does allow this tag to be missing in conway.
```suggestion
      , a (maybe_tagged_set committee_cold_credential)
```